andela/ah-92explorers-frontend
# #166789947 Update TravisCI To Run Tests

#### What does this PR do?
Update TravisCI to run tests

#### Description of Task to be completed
- add .travis.yml
- collect TravisCI markdown badge
- add the badge in readme.md

#### How should this be manually tested?
1. Run the following commands in the terminal

```
git clone https://github.com/andela/ah-92explorers-frontend.git
cd ah-92explorers-frontend
git checkout ch-update-travisci-166789947
npm install
```

2. Go to the readme, you will see TravisCI badge

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#166789947](https://www.pivotaltracker.com/story/show/166789947)

#### Screenshots (if appropriate)
![github com](https://user-images.githubusercontent.com/25223709/63112364-57c16a00-bf90-11e9-88af-02ed5996d022.jpg)


#### Checklist
